src/App.js

```diff
@@ -1,17 +1,115 @@
-import SearchBar from './components/SearchBar/SearchBar';
-import Users from './components/Users/Users';
-import './App.css';
+import React, { useEffect, useState } from "react";
 
+import SearchBar from "./components/SearchBar/SearchBar";
+import Users from "./components/Users/Users";
+import Loading from "./components/loading/Loading";
+import Error from "./components/error/Error";
+import "./App.css";
+
+const API = "https://users-app-backend.onrender.com/users";
```

Comment:
You've added `.env` and `dotenv` so I think you can have this `API` constant use an environment variable.